techmatters/terraso-backend
# feat: Story map membership pending

## Description


### Checklist



### Related Issues
https://github.com/techmatters/terraso-product/issues/336

### Tests to implement
- [x] Test register pending membership with user
- [x] Test Collaboration GQL nodes and filters
- [x] Test Stroy map filter sets for emails
- [x] Test story map configuration and membership list gql restriction
- [x] Test save membership
	- Check permissions
	- Check if it is for story map
- [x] Test approve membership with token
	- Existing user
	- Not existing user
- [x] Test approve membership
	- Only your own
- [x] Test delete mutation
	- Your own
	- Manager
